apps/trading/e2e/actions/utils.py

```diff
@@ -46,3 +51,22 @@ def change_keys(page: Page, vega:VegaServiceNull, key_name):
     page.get_by_test_id("key-" + vega.wallet.public_key(key_name)).click()
     page.click(f'data-testid=key-{vega.wallet.public_key(key_name)} >> .inline-flex')
     page.reload()
+
+def element_contains_text(locator: Locator, expected_pattern, timeout=5):
```

Comment:
@dalebennett1992 why exactly do we need this function? Does playwright not have some method for waiting for elements to appear?

Reply:
Quick search in the docs and found a `wait_for_selector` method can we use that here?

Reply:
so this was because the updated figures are not immediately available. The element is available and easy to locate but I wanted to wait for it contain the updated figures. I could have waited for the element containing the given text, but the playwright would return the element handle. I wanted the bool for the sake of having something nice and testable instead of a timeout or something. I can change though you do not agree